core/jazz_logs/config/prod-config.json

```diff
@@ -1,5 +1,6 @@
 {
-  "BASE_URL": "https://{inst_elastic_search_hostname}/",
+  "BASE_URL": "{inst_elastic_search_hostname}/",
```

Comment:
I don't see `BASE_URL` being used anymore. Remove if not being used.

Reply:
Done